pages/common/arduino.md

```diff
@@ -26,3 +26,11 @@
 - Save any (changed) preferences to `preferences.txt`:
 
 `arduino --save-prefs`
+
+- Install latest SAM board support:
+
+`arduino --install-boards "arduino:sam"`
```

Comment:
`{{...}}` can be used to underline that `--install-boards` accepts any valid value and not just `arduino:sam`:

```suggestion
`arduino --install-boards "{{arduino:sam}}"`
```

Reply:
@EmilySeville7cfg, requested changes have been updated!